open_spiel/games/crazy_eights_test.cc

```diff
@@ -0,0 +1,45 @@
+// Copyright 2023 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+
+#include "open_spiel/games/crazy_eights.h"
+
+#include "open_spiel/abseil-cpp/absl/strings/str_format.h"
+#include "open_spiel/spiel.h"
+#include "open_spiel/tests/basic_tests.h"
+
+
+namespace open_spiel {
+namespace crazy_eights {
+namespace {
+
+void BasicGameTests() {
```

Comment:
Not strictly necessary but if possible I'd suggest doing a test of a couple scenarios where things are likely to break (e.g. skipping players or drawing two) and make sure everything works as expected, especially the observation string/tensor.